azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.md

```diff
@@ -23,7 +23,7 @@ Retrieves the list of owners for an application object.
 
 ### Example 1: Get the owner of an application
 ```
-PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -Top 1
+PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Top 1
```

Comment:
Object ID not updated in the output.

Reply:
Changes implemented here are ok for the purposes of this PR